src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -480,7 +480,7 @@ void InitSurfaceHeatBalance(EnergyPlusData &state)
 
         if (state.dataEnvrn->SunIsUp && state.dataDaylightingData->ZoneDaylight(NZ).TotalDaylRefPoints != 0) {
             if (state.dataHeatBalSurfMgr->InitSurfaceHeatBalancefirstTime) DisplayString(state, "Computing Interior Daylighting Illumination");
-            DayltgInteriorIllum(state, NZ);
+            if (state.dataDaylightingManager->maxNumRefPtInAnyZone > 0) DayltgInteriorIllum(state, NZ);
```

Comment:
@matthew-larson This check is fine, but I don't understand how it's changing anything. If `maxNumRefPtInAnyZone` is the max of all `ZoneDaylight(NZ).TotalDaylRefPoints` and it's zero, that says this line won't be executed because of the `if ZoneDaylight(NZ).TotalDaylRefPoints !-0` above.  Also, the PR description says "since the zone has no windows and thus no solar info". Every zone should be part of a solar enclosure, whether it has windows or not. What am I missing here?